`make install` should only ever copy files

Several times now i encountered an issue with `make install` doing other things than just copying files. For example: sometimes it would try to compile rust binaries even if you just ran `uv run make` or now it would try this in the nightly plugins repo CI:

```
mkdir -p /usr/local/share/man/man7
mkdir -p /usr/local/share/man/man8
mkdir -p /usr/local/share/doc/c-lightning
wiregen wire/peer_wiregen.h
Traceback (most recent call last):
  File "/home/runner/work/plugins/plugins/lightning/tools/generate-wire.py", line 27, in <module>
    from mako.template import Template
ModuleNotFoundError: No module named 'mako'
make: *** [Makefile:359: wire/peer_wiregen.h] Error 1
rm wire/peer_wiregen.h
```

This all comes from the fact that both rust and uv are often installed per user and `make install` is often called with `sudo` which will have a different environment. So if possible `make install` should not run any python scripts that require dependencies or compile anything ever.

--- a/wire/Makefile
+++ b/wire/Makefile
@@ -109,8 +109,6 @@ wire/channel_type_wiregen.c_args := $(wire/channel_type_wiregen.h_args)
 WIRE_GENERATED := $(filter %printgen.h %printgen.c %wiregen.h %wiregen.c, $(WIRE_SRC) $(WIRE_HEADERS))
 $(WIRE_GENERATED): wire/Makefile
 
-# You don't need to keep these if they don't exist.
-.INTERMEDIATE: $(WIRE_GENERATED)
 
 maintainer-clean: wire-maintainer-clean
 

